zabbix_metrics_watcher.py

```diff
@@ -176,24 +181,15 @@ def _send_zabbix_message(zabbix_host, osbs_master, key, value, print_command=Tru
 
 def filter_completed_builds(completed_builds):
     # Remove all completed_builds which are not within this hour
-    now = int(time())
-    return {k: v for k, v in completed_builds.items() if (now - v) < 3600}
-
-
-def heartbeat(zabbix_host, osbs_master):
-    while True:
-        _send_zabbix_message(zabbix_host, osbs_master,
-                             "heartbeat", int(time()), print_command=False)
-        sleep(10)
+    return {k: v for k, v in completed_builds.items()
+            if (datetime.datetime.utcnow() - v.replace(tzinfo=None)).total_seconds() < 3600}
```

Comment:
Openshift returns date in UTC as well, but Python cannot calculate size delta if one of the objects is time aware and the other one is not. I'll try `.now()` approach though